app/src/main/java/com/example/android/persistence/codelab/step3/BooksBorrowedByUserActivity.java

```diff
@@ -16,31 +16,44 @@
 
 package com.example.android.persistence.codelab.step3;
 
+import android.arch.lifecycle.ViewModelProviders;
 import android.os.Bundle;
 import android.support.annotation.NonNull;
+import android.support.v7.app.AppCompatActivity;
 import android.view.View;
 import android.widget.TextView;
 
-import android.arch.lifecycle.LifecycleActivity;
-import android.arch.lifecycle.ViewModelProviders;
 import com.example.android.codelabs.persistence.R;
 import com.example.android.persistence.codelab.db.Book;
 
 import java.util.List;
 
-public class BooksBorrowedByUserActivity extends LifecycleActivity {
+public class BooksBorrowedByUserActivity extends AppCompatActivity {
 
     private BooksBorrowedByUserViewModel mViewModel;
 
     @SuppressWarnings("unused")
     private TextView mBooksTextView;
 
+    @SuppressWarnings("unused")
+    private static void showBooksInUi(final @NonNull List<Book> books,
```

Comment:
why is this a `private static` method? can it be just private?

Reply:
I have absolutely no idea. Changed to non-static.